03.create/04.entities/14.primitive-painting-set-tutorial/docs.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Tutorial: Build a Painting Set
+taxonomy:
+    category: docs
+---
+
+There are different ways to create games and experiences in High Fidelity. In this tutorial, we'll cover how to make a primitive pixel-like painting set to allow users to "paint" pictures on a canvas made out of "pixels" that are box entities.
+
+**On This Page:**
+
+- [Prerequisites](#prerequisites)
+- [Create a Painting Set](#create-a-painting-set)
+- [Add a Paint Brush Script](#add-a-paint-brush-script)
+
+
+## Prerequisites
+
+Consider getting familiar with the following concepts before starting this tutorial:
+
+- [Create New Entities](../create-entities)
+- [Change How Entities Look](../entity-appearance)
+- [Define an Entity's Behavior](../entity-behavior)
+- [Get Started with Scripting](../../../script/get-started-with-scripting)
+- [Client Entity Scripts](../../../script/client-entity-scripts)
+- [Write Your Own Scripts](../../../script/write-scripts)
+
+## Create a Painting Set
+
+Our painting set comprises three elements:
+* [A brush, made of a cylinder and a sphere object](#create-a-paint-brush).
+* [A palette, made of an octagon and sphere object](#create-a-paint-palette).
+* [A canvas, made of box entities](#create-a-pixel-canvas).
+
+All of the logic for our painting set is contained in the brush head. The rest of the content is made by parenting primitives to one another to make our brush, palette, and canvas.
```

Comment:
I don't think we use the word "primitives" anywhere else in the help. What is it?

Reply:
building blocks/ entities. I'll change it to entities

Reply:
basically the most basic entities